src/glossary.ts

```diff
@@ -1,28 +1,11 @@
 import type { HeadersObject, Headers } from 'headers-polyfill'
-import type { LazyCallback } from './utils/createLazyCallback'
+import type { InteractiveIsomorphicRequest } from './InteractiveIsomorphicRequest'
```

Comment:
We used to export the `InteractiveIsomorphicRequest` type from the root of the package as well, as it was in the glossary module before. Could you please add the IIR export from its own module in the `index.ts`?

Reply:
Done